src/main/kotlin/com/jobtracker/api/controllers/models/ContactModel.kt

```diff
@@ -0,0 +1,24 @@
+package com.jobtracker.api.controllers.models
+
+import com.jobtracker.api.repository.entities.ApplicationEntity
+import com.jobtracker.api.repository.entities.ContactEntity
+import com.jobtracker.api.repository.entities.CompanyEntity
+import org.apache.commons.lang3.mutable.Mutable
+import java.util.UUID
+
+data class ContactModel(
+  var contactId: UUID = UUID.randomUUID(),
+  var companyObject: CompanyEntity,
+  val fullName: String,
+  val positionTitle: String,
+  val emailAddress: String,
+  val phoneNumber: String,
+  val notes: String,
+  val applications: MutableList<ApplicationEntity>
```

Comment:
This would need to be a list of `applicationId`'s

Reply:
Same comment as above. Not sure about what to do about the type situation here.